src/trading/place-trade.js

```diff
@@ -28,34 +28,24 @@ var constants = require("../constants");
  * @param {function} p.onSuccess Called when the full trade completes successfully.
  * @param {function} p.onFailed Called if any part of the trade fails.
  */
+
 function placeTrade(p) {
   var normalizedPrice = normalizePrice({ minPrice: p.minPrice, maxPrice: p.maxPrice, price: p.limitPrice });
   var numTicks = new BigNumber(p.maxPrice, 10).minus(new BigNumber(p.minPrice, 10)).dividedBy(new BigNumber(p.tickSize, 10)).toFixed();
+  var orderType = (["buy", "sell"])[p._direction];
   getBetterWorseOrders({
     marketID: p._market,
     outcome: p._outcome,
-    amount: p.amount,
+    orderType,
```

Comment:
Should we make getBetterWorse accept the 0/1 ? we already had some heartache with bid/ask buy/sell

Reply:
Hm, I'd say keep it as "buy"/"sell" for augur-node, since that's what's used everywhere in augur-node -- best to be consistent IMO